tutorials/p5js/_posts/2017-09-16-which-processing.md

```diff
@@ -38,9 +38,9 @@ To understand how all of this fits together, let's see a timeline of the major e
   - Java had a goal of ["write once, run anywhere"](https://en.wikipedia.org/wiki/Write_once,_run_anywhere) - any computer that can run Java can run programs written in Java. This was a big change from other programming languages at the time, which had to be recompiled for each type of computer.
 
 - **2001:** Processing is developed by an artist named [Casey Reas](https://en.wikipedia.org/wiki/C.E.B._Reas) and a data scientist named [Ben Fry](https://en.wikipedia.org/wiki/Benjamin_Fry).
-  - Processing is built on top of Java, with the goal of providing a simplified syntax that allows developers to make creative animated and interactive applications.
+  - Processing is built on top of Java, with the goal of providing a simplified syntax that lets developers to make creative animated and interactive applications.
```

Comment:
Thanks for making this change!

In these cases where you change "allows" to "lets", can you also get rid of the extra "to"?

In other words:

"...syntax that lets developers to make creative..."

Should be:

"...syntax that lets developers make creative..."

(Same thing in the next change.)